app/assets/javascripts/discourse/app/controllers/user-private-messages.js

```diff
@@ -57,21 +48,18 @@ export default Controller.extend({
     );
   },
 
-  actions: {
-    changeGroupNotificationLevel(notificationLevel) {
-      this.group.setNotification(notificationLevel, this.get("user.model.id"));
-    },
-    archive() {
-      this.bulkOperation("archive_messages");
-    },
-    toInbox() {
-      this.bulkOperation("move_messages_to_inbox");
-    },
-    toggleBulkSelect() {
-      this.toggleProperty("bulkSelectEnabled");
-    },
-    selectAll() {
-      $("input.bulk-select:not(checked)").click();
-    },
+  @action
+  changeGroupNotificationLevel(notificationLevel) {
+    this.group.setNotification(notificationLevel, this.get("user.model.id"));
+  },
+
+  @action
+  toggleBulkSelect() {
+    this.toggleProperty("bulkSelectEnabled");
+  },
+
+  @action
+  selectAll() {
+    $("input.bulk-select:not(checked)").click();
```

Comment:
Can we move this to a component? We should not be manipulating the dom in controllers.